ArcGISRuntimeSDKQt_QMLSamples/Geometry/ClipGeometry/README.md

```diff
@@ -0,0 +1,26 @@
+# Clip Geometry
+
+This sample demonstrates how to clip a geometry with an envelope using the GeometryEngine.
+
+![](screenshot.png)
+
+Click or tap the "Clip" button to clip the blue graphic with the red envelopes.
+
+## How it works
+1. Create a `graphicsOverlay` containing an envelope.
+2. Create another that contains three other envelopes: one that intersects a part of the first graphic, one that is entirely within it, and another completely outside of it.
+3. Create one more `graphicsOverlay` that contains no graphics.
+4. Create a button object that, when clicked, calls a function that hides the main graphic *only*.
+5. In the same function, iterate through the overlay with the three envelopes. `forEach` one, a `Graphic` object is created by calling `GeometryEngine.clip` method, passing in the main graphic and the current envelope in the iteration as arguments.
```

Comment:
`...passing in the main graphic's geometry...`

`...current envelope as arguments.`.  iteration statement not needed, it's implied there's a loop involved via the `forEach`.